lib/createInlinePluginCreator.js

```diff
@@ -215,20 +201,12 @@ function createInlinePluginCreator(packages, multiContext, synchronizer, flags)
 			debug("notes generated: %s", pkg.name);
 
 			// Return the notes.
-			return notes.join("\n\n");
+			return notes.join("\n").replace("\n\n", "\n").trim();
```

Comment:
This may not be needed, but I was noticing that our release notes contained a lot of unnecessary new lines